### ISSUE TITLE
How to get dockerVersion or dockerInfo runinng?

### ISSUE THREAD

**Issue by staleks** [bmuschko/gradle-docker-plugin#34]
So far I've followed example and suggested steps:

1. added 

buildscript {
    repositories {
        jcenter()
    }

    dependencies {
        classpath 'com.bmuschko:gradle-docker-plugin:2.0.2'
    }
}

at the begining of build.gradle file

2. applied both plugins:

apply plugin: 'com.bmuschko.docker-remote-api' and
apply plugin: 'com.bmuschko.docker-java-application'

Now if I go to command line, and type
$ gradle tasks

I have just these three at exposal:

Docker tasks
------------
dockerBuildImage - Builds the Docker image for the Java application.
dockerCopyDistResources - Copies the distribution resources to a temporary directory for image creation.
dockerPushImage - Pushes created Docker image to the repository.

What about dockerImage or dockerVersion?

How to run those ?

Thank You

**Comment by bmuschko** [comment]
Neither of these plugins pre-defines the tasks `dockerImage` or `dockerVersion`. You will need to set them up yourself.

**Comment by staleks** [comment]
Ok, thnx.

You are 100% right.

I've added following:

1. at the begining of the build.gradle file I had to import:

import com.bmuschko.gradle.docker.tasks.*

2. defined task dockerInfo, e.g.

task dockerInfo(type: DockerInfo) {
	doLast {
		println "dumping DockerInfo"
	}	 	
}



NOTE: As I use Spring Framework a lot, I've disabled commons-logging, so I was getting following error:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':rest-server:dockerInfo'.
> org/apache/commons/logging/LogFactory

Please have this in mind

**Comment by bmuschko** [comment]
I don't understand your second comment about disabling `commons-logging`. Can you clarify and provide the stack trace?

**Comment by staleks** [comment]
You don't have to worry about this. This is something specific in my case.

I will try to explain, briefly :)

I have Gradle multi-project setup.

As I use a SpringFramework, in parent project, I have disabled `commons-logging` for all projects. E.g.

	configurations.all {
		exclude group: 'commons-logging'
	}

As suggested in Spring Framework documentation, commons-logging should be excluded. And I am using slf4j with logback combination.

Now, as I've disabled `commons-logging` in general for my projects, I was getting already mentioned error.

But actual error was hiddent that there is no Docker API at localhost:2375 to respond about DockerInfo.

After I re-enabled `commons-logging` error is:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':rest-server:dockerInfo'.
> org.apache.http.conn.HttpHostConnectException: Connect to localhost:2375 [localhost/127.0.0.1] failed: Connection refused

I hope that this will clarify things for future developers facing same
